CMakeLists.txt

```diff
@@ -446,10 +450,14 @@ function(ADD_WINDOWS_DLL_PATH_TO_TEST TEST_NAME)
 endfunction()
 
 # --- MAIN LIBRARY -------------------------------------------------------------
-add_subdirectory(src)
+if(NOT BUILD_STANDALONE_PYTHON_INTERFACE)
```

Comment:
I would add a variable called: BUILD_MAIN_PROJECT_AND_ASSOCIATED_OUPUTS

Reply:
Do you mean something like this ?
```cmake
set(BUILD_MAIN_PROJECT_AND_ASSOCIATED_OUPUTS (NOT BUILD_STANDALONE_PYTHON_INTERFACE))
```